README.md

```diff
@@ -0,0 +1,158 @@
+# E-Commerce Project For Baby Tools
+A small project with basic CRUD operations. In this application you can create baby-product listings and sort them in the admin area. The app was built with Django, which is based on Python and lets you spin up the project quickly.
+
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+1. [Hints](#hints)
+1. [Download Repository](#download-repository)
+1. [Install Docker](#install-docker)
+1. [Create Dockerfile](#create-dockerfile)
+1. [Build Docker Image](#build-docker-image)
+1. [Start Docker Container](#start-docker-container)
+1. [How to connect](#how-to-connect)
+1. [Photos](#photos)
+1. [Useful commands](#useful-commands)
+
+
+## Prerequisites
+
+- Python 3.9
+- Django 4.0.2
+- Venv
+- Docker
+- git
+
+---
+
+## Hints
+This section will cover some hot tips when trying to interacting with this repository:
+
+- Settings & Configuration for Django can be found in `babyshop_app/babyshop/settings.py`
+- Routing information, such as available routes can be found from any `urls.py` file in `babyshop_app` and corresponding subdirectories
+
+---
+
+## Download Repository
```

Comment:
Rename it to `Quickstart` and do not forget to update the Table of contents sections.